cratonclient/tests/base.py

```diff
@@ -15,9 +15,31 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+import sys
+import six
 from oslotest import base
 
+from cratonclient.shell import main
 
 class TestCase(base.BaseTestCase):
 
     """Test case base class for all unit tests."""
+    def shell(self,arg_str, exitcodes=(0,)):
```

Comment:
You should have a base test case for all the shell tests, so no this does not belong here.